reconcile/openshift_resources.py

```diff
@@ -172,6 +177,22 @@ def fetch_provider_vault_secret(path, version, name, labels, annotations):
     return openshift_resource
 
 
+def fetch_provider_route(path, tls_path, tls_version):
+    openshift_resource = fetch_provider_resource(path)
+
+    if tls_path is None or tls_version is None:
+        return openshift_resource
+
+    tls = {}
```

Comment:
ah, so you are replacing whatever is in tls. would it make sense to merge instead?

Reply:
i expect nothing to be present in `tls`, as it is secret data

Reply:
ok, this should be documented

Reply:
(I don't mean in the code)

Reply:
it will :)